styles/_notifications.scss

```diff
@@ -112,8 +112,8 @@
     display: none;
   }
 
-  &.notification-error::before {
-    @extend .notification-icon::before;
+  &.notification-error, ::before {
```

Comment:
This seems to be causing the notification error to render weirdly:

<img width="701" alt="Screenshot 2023-03-13 at 11 43 26" src="https://user-images.githubusercontent.com/1172767/224679329-a5b68449-8db9-41fb-8ce7-35ab5e600e3d.png">

This seems to fix it:

```suggestion
  &.notification-error::before {
```

Reply:
This is a great catch!
I did test this one (though only in the request form) and it's a bit scary that I didn't catch the same bug.
Thanks so much for also looking into it 🙏🏼 
I'll address the issue and try to understand why I missed it in the first place.